File: app/src/main/java/com/rafaelmfer/nasaexperience/AsteroidModel.java
package com.rafaelmfer.nasaexperience;

public class AsteroidModel {

    private String asteroidNumber, asteroidDescription;

    public AsteroidModel() {
    }

    public AsteroidModel(String asteroidNumber, String asteroid) {
        this.asteroidNumber = asteroidNumber;
        this.asteroidDescription = asteroid;
    }

    public String getAsteroidNumber() {
        return asteroidNumber;
    }

    public void setAsteroidNumber(String asteroidNumber) {
        this.asteroidNumber = asteroidNumber;
    }

    public String getAsteroidDescription() {
        return asteroidDescription;
    }

    public void setAsteroidDescription(String asteroidDescription) {
        this.asteroidDescription = asteroidDescription;
    }
}

File: app/src/main/java/com/rafaelmfer/nasaexperience/model/AsteroidModel.kt
package com.rafaelmfer.nasaexperience.model

import androidx.room.ColumnInfo
import androidx.room.Entity
import androidx.room.PrimaryKey

@Entity(tableName = "asteroid_model")
data class AsteroidModel(
        @PrimaryKey(autoGenerate = true)
        var id: Int,

        @ColumnInfo(name = "asteroid_number")
        val asteroidNumber: String,

        @ColumnInfo(name = "asteroid_description")
        val asteroidDescription: String
)